tenzir.yaml.example

```diff
@@ -38,7 +47,7 @@ tenzir:
   # Defaults to one of the following paths, selecting the first that is
   # available:
   #   - $LOGS_DIRECTORY/server.log
-  #   - <db-directory>/server.log
+  #   - <state-directory>/server.log
```

Comment:
Might be a follow-up, but IIRC we also wanted something like a `log-directory`?

Reply:
Ideally yes, but I'm not sure it's worth the effort of figuring out how to make that backwards-compatible with `tenzir.log-file`. Having a clearer split between persistent state and cache directories was something we actually have a need for.

Reply:
Would it then make sense to move the logs over into the cache directory, given that they are non-essential?

Reply:
I think they are very much essential.

Reply:
I don't have a strong opinion here. I would imagine that users that care about the logs feed them into their o11y tool of choice anyway.

Reply:
Some day we'll have the `logs` operator and then users can do whatever they want with it.

Reply:
Hmmm, maybe. Before that, we may need an OTel logging sink and adopt structured logging.